src/libraries/Microsoft.PowerFx.Core/Public/IPowerFxScope.cs

```diff
@@ -21,5 +21,12 @@ public interface IPowerFxScope
         /// Provide intellisense for expression
         /// </summary>
         IIntellisenseResult Suggest(string expression, int cursorPosition);
+
+        /// <summary>
+        /// Provider quick fix suggesions.
+        /// </summary>
+        /// <param name="expression">The formula expression.</param>
+        /// <returns>Collection of quick fixes.</returns>
+        CodeActionResult[] QuickFix(string expression);
```

Comment:
Adding a new method to an interface is a breaking change. 
Can you put on a new interface (like what we did for IPowerFxScopeDisplayName)?

That also gives a clear way for things that don't provide this feature - they just don't implement the interface.